src/utils/request/onUnhandledRequest.ts

```diff
@@ -116,6 +116,17 @@ ${handlers.map((handler) => `  • ${handler.info.header}`).join('\n')}`
   return `Did you mean to request "${handlers[0].info.header}" instead?`
 }
 
+export function tryAndSilenceIfThrows<Payload>(
```

Comment:
Please, would you be comfortable writing a few unit tests for this helper function? 
Let me know if you need any help, I'd be glad to help you with the tests.